cmd/geth/chaincmd.go

```diff
@@ -242,10 +242,9 @@ func importChain(ctx *cli.Context) error {
 	// Start system runtime metrics collection
 	go metrics.CollectProcessMetrics(3 * time.Second)
 
-	stack, _ := makeConfigNode(ctx)
+	stack, _, chain, db := makeFullNode_(ctx)
```

Comment:
seems kinda of hacky, since you're actually starting a node (opening all ports, etc) by calling this.

Reply:
I would say less hacky than using a fake fake engine though.

Reply:
I agree. But the config needed to properly set up the engine is all over the place right now. We should probably create an issue to tidy that part up.